test/isIsogram.test.js

```diff
@@ -0,0 +1,46 @@
+import { equal } from 'assert';
+import { isIsogram } from '../isIsogram.js';
+
+it('Is an Isogram - Dermatoglyphics', () => {
+    const result = isIsogram('Dermatoglyphics');
+    const expectedResult = true;
+    equal(result, expectedResult);
+});
+
+it('Is an Isogram - ant', () => {
+    const result = isIsogram('ant');
+    const expectedResult = true;
+    equal(result, expectedResult);
+});
+
+it('Not an Isogram - aba', () => {
+    const result = isIsogram('aba');
+    const expectedResult = false;
+    equal(result, expectedResult);
+});
+
+it('Not an Isogram, different casing - Aba', () => {
+    const result = isIsogram('Aba');
+    const expectedResult = false;
+    equal(result, expectedResult);
+});
+
+it('Not an Isogram, same characters next to each other - goose', () => {
+    const result = isIsogram('goose');
+    const expectedResult = false;
+    equal(result, expectedResult);
+});
+
+it('Not an Isogram, same characters next to each other, different casing - GoOse', () => {
+    const result = isIsogram('GoOse');
+    const expectedResult = false;
+    equal(result, expectedResult);
+});
+
+//As per the descrription in the instructions, we assume an empty string is an isogram 
```

Comment:
```suggestion
//As per the description in the instructions, we assume an empty string is an isogram 
```

did you know you could do that as part of the PR?
![image](https://github.com/kirithecat/MoniquesPlaygroundOne/assets/4237268/618b311b-868f-4feb-b73b-e4069e1b3914)

Reply:
removed the comment, therefore, no more typo but it's cool that you can commit and make the change straight from Github